packages/modular-scripts/src/start.ts

```diff
@@ -22,6 +22,7 @@ export default async function start(appPath: string): Promise<void> {
     env: {
       USE_MODULAR_BABEL: process.env.USE_MODULAR_BABEL,
       MODULAR_ROOT: modularRoot,
+      SKIP_PREFLIGHT_CHECK: 'true',
```

Comment:
It seems a bit premature to set this to auto skip now when we haven't taken over build config and still using react-scripts. We can wait until there is a reason to skip preflight checks to set this

Reply:
We already call it directly ourselves in `preFlightCheck ` currently - this moves it directly into our startup checks so the gap between react-scripts and the `start` / `build` APIs are smaller for taking control of the build directly.